test/unit/logic/transaction.js

```diff
@@ -1078,24 +1078,24 @@ describe('transaction', function () {
 		});
 
 		it('should return transaction object with data field', function () {
-			var rawTrs = _.cloneDeep(rawTransaction);
-			var trs = transactionLogic.dbRead(rawTrs);
+			var __rawTransaction = _.cloneDeep(rawTransaction);
+			var __transaction = transactionLogic.dbRead(__rawTransaction);
 
-			expect(trs).to.be.an('object');
-			expect(trs.asset).to.have.property('data');
+			expect(__transaction).to.be.an('object');
+			expect(__transaction.asset).to.have.property('data');
 		});
 
 		it('should return null if id field is not present', function () {
-			var rawTrs = _.cloneDeep(rawTransaction);
-			delete rawTrs.t_id;
+			var __rawTransaction = _.cloneDeep(rawTransaction);
```

Comment:
Why not `rawTransactionClone`? We need to put away `__` and `_`:
```javascript
// bad
this.__firstName__ = 'Panda';
this.firstName_ = 'Panda';
this._firstName = 'Panda';

// good
this.firstName = 'Panda';
```

Reply:
I agree.